services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,36 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
+  BaseType_t queueSend = xQueueSend(thermalMgrQueueHandle, event, 0);
```

Comment:
- [ ] probably best to put this line inside of the if check
- [ ] check that the queue handle isn't null 
- [ ] check that the event isn't null